src/pybind/mgr/dashboard/tests/test_access_control.py

```diff
@@ -652,6 +669,7 @@ def test_load_v1(self):
                         "username": "admin",
                         "password":
                 "$2b$12$sd0Az7mm3FaJl8kN3b/xwOuztaN0sWUwC1SJqjM4wcDw/s5cmGbLK",
+                        "pwdExpirationDate": null,
```

Comment:
In this test you are testing the upgrade from V1 to V2, so you should not edit the V1 config by manually adding `pwdExpirationDate` setting here. This is done by the upgrade method.